public/New-DbaConnectionString.ps1

```diff
@@ -196,7 +196,6 @@ function New-DbaConnectionString {
         [string]$ClientName = "custom connection",
         [int]$ConnectTimeout,
         [string]$Database,
-        [ValidateSet('Mandatory', 'Optional', 'Strict', 'True', 'False')]
         [switch]$EncryptConnection = (Get-DbatoolsConfigValue -FullName 'sql.connection.encrypt'),
```

Comment:
old configs may return a non bool/switch. we should set it after, in the beginning/body

Reply:
`[switch]$EncryptConnection = (Get-DbatoolsConfigValue -FullName 'sql.connection.encrypt'),` is exactly the same line as in `Connect-DbaInstance`. We should have seen complaints if this would be a problem. We should definitly keep the parameter definition of these two commands as similar as possible.